pkgs/top-level/all-packages.nix

```diff
@@ -9116,6 +9116,8 @@ with pkgs;
 
   mob = callPackage ../applications/misc/mob {};
 
+  mopac = callPackage ../applications/science/chemistry/mopac { };
```

Comment:
I think there is a "chemistry" section  in the top level file.

Reply:
Uh, nice. Completely missed it. `all-packages.nix` is crazy :)